daemons/ipa-kdb/ipa_kdb_kdcpolicy.c

```diff
@@ -0,0 +1,107 @@
+/* 
+ * MIT Kerberos kdcpolicy plugin for FreeIPA
+ *
+ * Authors: Robbie Harwood <rharwood@redhat.com>
+ *
+ * Copyright (C) 2018 Red Hat, Inc.
+ * see file 'COPYING' for use and warranty information
+ *
+ * This program is free software you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <errno.h>
+#include <syslog.h>
+#include <krb5/kdcpolicy_plugin.h>
+
+#include "ipa_krb5.h"
+#include "ipa_kdb.h"
+
+struct krb5_kdcpolicy_moddata_st {
+    ;
+};
```

Comment:
An empty structure? Is this some krb5-ism?

Reply:
Ah, no.  It's places where we could store IPA state.  I'll add a comment.